app/Http/Controllers/BookController.php

```diff
@@ -146,4 +146,40 @@ public function returnBook(int $id, Request $request)
             'success' => true
         ]);
     }
+    public function addBook(Request $request)
+    {
+        $request->validate([
+            'title' => 'required|string',
+            'author' => 'required|string',
```

Comment:
Just to be on the safe side, can you get some max length checks on this to line up with your db column lengths